code/parachain/frame/cosmwasm/src/types.rs

```diff
@@ -34,7 +34,11 @@ pub enum EntryPoint {
 	Execute,
 	Migrate,
 	Reply,
-	// TODO(hussein-aitlahcen): do we want to support cosmwasm sudo?
+	IbcChannelOpenCall,
```

Comment:
For these, for the sake of consistency, we should remove "Call" and make it like `IbcChannelOpen`, etc.

Reply:
just for you, i just shown weirdness of API. why would one cp paste same thing many time?

Reply:
api is and was overegnineered